change/react-native-windows-b7f2b1f4-dbd1-4e33-9d88-554afde1b11a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Include RuntimeExecutor.h in nuget",
+  "packageName": "react-native-windows",
```

Comment:
Food for thought, I wonder if there are ways we can consolidate Office DLL specific logic to separate JS package.